### DISCUSSION TITLE
Apple Vision Pro

### DISCUSSION THREAD

**Discussion by William08290829** [1101-GS4719-C/Python-Community-GS4719-C#1919] [General]
去年 6 月初，蘋果發表了首款 VR 頭戴裝置--Apple Vision Pro，產品也在今年2/2 於美國正式開賣，而其實該產品本質上是一款可穿戴式 MacBook。由於身邊有朋友從剛出的時候就一直說要買，所以我也有稍微關注了一下，~~但不多~~

我看了網路上蠻多的影片還有文章關於這次Apple這次新花樣的開箱，老實說看到影像的呈現方式還有畫質真的是蠻被驚豔到的。
大概是這些文章影片:
[Vision Pro使用心得：實測一致獲好評！好在哪？一台10萬值得嗎？盤點評價](https://www.bnext.com.tw/article/78183/vision-pro-testing?)

[Apple Vision Pro入手！3C達人開箱心得老實說：畫質與體驗都是目前最強](https://udn.com/news/story/7270/7777605)

[帶你穿越到未來！Vision Pro詳細開箱！](https://www.youtube.com/watch?v=t3YaLnLMgmg)

[【長期心得】Apple Vision Pro 該不該買？](https://www.youtube.com/watch?v=u8UayRMeR-c)

看完這些發現大概網路上都覺得有共同的兩項缺點:

1. 價格不斐，目前要價約3499 美元 (約台幣10萬多)
2. 重量不輕，約600多克，長時間待著臉會產生不適，因為等於拿著一台iPad pro貼在臉上戴

但除了這兩項缺點，基本上驚豔的程度我覺得還是蠻大的
儘管存在這兩個缺點，網路上對於這款產品的驚豔程度仍然相當高，畢竟目前只是第一代產品。它成功地將虛擬UI與現實世界完美結合，具有精確穩定的空間定位，畫面上感覺是真的蠻美的。
不知道大家是怎麼想的，歡迎討論，但我是覺得如果拿掉第一項缺點我可能就會買了 σ`∀´)σ
~~還是有好野人可以讓我體驗看看~~

**Comment by KH1126** [comment]
據說還需要隨身帶行動電源，來維持電量，
這重量也不算輕。

此外，
即使Apple Vision Pro 價格不斐，銷量可能因為價格不會太好，
但我認為這個產品是很重要的，可算是一個里程碑產品。

**Comment by jylin34** [comment]
不久之前看到這部Vision Pro相關的影片，我覺得講得很好: [ Vision Pro 真正的顛覆之處：交互設計師 + 開發者的專業深度解讀](https://www.youtube.com/watch?v=m91US-6qfUo)
Vision Pro厲害的不僅僅是他的硬體規格，而是他的應用開發門檻遠低於其他市場上的XR競品，VisionOS可以直接兼容ios的App，意味著開發者可以用熟悉的技術、語言直接開始開發VisionOS應用，相比Meta Quest, Pico這類競品，則需要去額外學習Unity等遊戲引擎去開發。

硬體問題還可以靠時間解決，但軟體、生態鏈沒有一開始設計好，後面可能就很難了。 可能 3499美金 有很大一部分就是體現在這XD

**Reply by wanderer657** [reply]
看來Apple Vision Pro的推出確實引起了不少關注。我也有稍微關注了一下，看了一些相關的影片和文章，對於它的畫質和呈現方式感到相當驚豔。

雖然網路上普遍對這款產品的兩個缺點——高昂的價格和相對較重的重量——提出了批評，但我仍然覺得它的驚豔程度相當大。它成功地將虛擬UI與現實世界完美結合，畫面上感覺非常真實美麗。

儘管目前只是第一代產品，但我相信未來會有更多的改進和創新。如果價格能稍微下降，我也許會考慮入手一個。
